client/src/components/OrganizationCredentialSelector.tsx

```diff
@@ -122,7 +123,7 @@ export function CombinedSelector({ isExpanded }: { isExpanded: boolean }) {
   };
 
   const noCredentialsAvailable =
-    availableCredentials.length === 0 && tempOrgValue !== "" && !isLoading;
```

Comment:
Here before we had something like this:
```
const noCredentialsAvailable =
    availableCredentials.length === 0 && tempOrgValue !== "" && !credLoading;
 ```
 
 but `credLoading` was never being updated inside this function `fetchAvailableCredentials` so that's why it wasn't blinking

Reply:
sounds great for me it's a valid solution if you think there's something better on the future, cool! Great job, thanks!